Tests/MoyaTests/MoyaProviderSpec.swift

```diff
@@ -178,11 +178,11 @@ final class MoyaProviderSpec: QuickSpec {
         describe("a provider with delayed stubs") {
             var provider: MoyaProvider<GitHub>!
             var plugin: TestingPlugin!
-            let delay: TimeInterval = 0.5
+            let delayRange = 0.15...0.30
 
             beforeEach {
                 plugin = TestingPlugin()
-                provider = MoyaProvider<GitHub>(stubClosure: MoyaProvider.delayedStub(delay), plugins: [plugin])
+                provider = MoyaProvider<GitHub>(stubClosure: MoyaProvider.delayedStub(delayRange), plugins: [plugin])
```

Comment:
These should be new specs, so we test both paths.

Reply:
Yeah agree, though I didn't want to spend too much time in case it's not something we'd like to introduce so updated one to make sure it works.